drivers/wdm/audio/backpln/portcls/pin_wavert.cpp

```diff
@@ -307,9 +301,9 @@ CPortPinWaveRT::DeviceIoControl(
 
     switch (IoStack->Parameters.DeviceIoControl.IoControlCode)
     {
-        case IOCTL_KS_PROPERTY:
+       case IOCTL_KS_PROPERTY:
```

Comment:
add the initial space back in the beginning to fix the indentation level that your fix broke.

Reply:
Fixed.